doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -10,9 +10,58 @@
+- Image Signing
+- Role-base Access Control
+- Secret Managment
+- Kubernetes Cluster Multi-tenancy security
+  - Tenant can be distinct teams or workload types (Develpoment or Production) within an organisation, each with a namespace
+  - Tenant can also be per dedicated kubernetes cluster
+- Integration with other security ecosystem like monitoring and alerting tools
 
 <a name="6.2"></a>
-## 6.2 Heading
+##  6.2 Principles
+The following are core principles to consider when securing cloud native applications;
+
+- Deploy only secure applications and trusted codes
+- Only deploy applications from validated and verified images
+- Only deploy applications from trusted registries
+- Containers orchestration (Kubernetes) must be secure with administrative boundaries between tenants
+  - Use Namespaces to establish security boundaries between tenants
+  - Create and define cluster network policies
+  - Run a cluster-wide pod security policy
+  - Turn on Audit Logging
+  - Separate sensitive workloads using namespaces
+  - Secure tenant metadata Access
+- Network segmentation using security zoning and network standards must be followed when deploying containers 
+- The Host OS running the containers must be hardened
+- Use container-aware runtime defense tools
+- Enable Role-Based Access Control (RBAC)
 
 <a name="6.3"></a>
-## 6.3 Heading
+## 6.3 Security Perimeters
+When applications or workloads run on Kubernetes, there are several layers which come into picture that govern the security. Each of these layers need to be secured within their perimeters. The various layers that come into picture are:
+
+- Container Registry: A container registry is a repository to manage container images. The access to container registry needs to be secured in order to provide a controlled access.
+- Container Images: Stored instance of a container that holds a set of software needed to run an application. Before loading them to container registry, they need to be made secured by performing various checks like vulnerability analysis, scans etc. These should also be signed from trusted sources.
+- Containers: A lightweight and portable executable image that contains software and all of its dependencies. The containers need to be prevented from accessing the underlying OS like loading of kernel modules, mounting of directories of underlying OS etc and it must also be ensured that they don't run in priveleged mode..
+- Pods: A Pod represents a set of running containers on your cluster. Kuberenetes inherently offers pod security policies that define a set of conditions that a pod must run with in order to be accepted into the system. These policies help in ensuring the necessary checks for running the pods.
+- Kubernetes Node: A Kubernetes node in an unsecured boundary can lead to a potential threat to the running workloads. Such a node may be hardened in terms of security by disabling unused ports, prohibiting root access etc.
```

Comment:
Sugggest "... a node may be hardened in terms of security by ..." to "... a node should be hardened by ..."

Reply:
Fixed.